File: desafio-mobile-android/app/src/main/java/com/app/githubclient/MyApplication.java
package com.app.githubclient;

import android.app.Application;

import com.facebook.drawee.backends.pipeline.Fresco;

/**
 * Created by thaynan on 24/05/2016.
 */

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Fresco.initialize(this);
    }
}

File: Cart/app/src/main/java/br/com/cart/MyApplication.kt
package br.com.cart

import android.app.Application
import br.com.cart.di.DependencyModules
import org.koin.android.ext.android.startKoin

class MyApplication: Application() {

    override fun onCreate() {
        super.onCreate()

        startKoin(this, listOf(DependencyModules.appModule))

    }

}